docs/guides/getting_started.md

```diff
@@ -408,32 +410,19 @@ const links = $('a[href]')
     .map((i, el) => $(el).attr('href'))
     .get();
 
-const {protocol, hostname} = new URL(request.loadedUrl);
+const { hostname } = new URL(request.loadedUrl);
 const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
 
-const sameDomainLinks = absoluteUrls.filter(url => url.protocol === protocol && url.hostname.endsWith(hostname));
+const sameDomainLinks = absoluteUrls.filter(url => url.hostname.endsWith(hostname));
 
 // ...
 ```
 
-This includes subdomains. In order to pin the entire origin, simply compare the `url.origin` property instead:
+This includes subdomains. In order to filter the same origin, simply compare the `url.origin` property instead:
 
 ```js
-// At the top of the file:
-const { URL } = require('url');
-
-// ...
-
-const links = $('a[href]')
-    .map((i, el) => $(el).attr('href'))
-    .get();
-
-const {origin} = new URL(request.loadedUrl);
+const { origin } = new URL(request.loadedUrl);
 const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
-
-const sameDomainLinks = absoluteUrls.filter(url => url.origin === origin);
```

Comment:
I think you wanted to keep a different line here?

Reply:
Oh you're right, lemme fix this real quick...

Reply:
PTAL :)